awsconfigs/apps/pipeline/s3/sync.py

```diff
@@ -345,6 +347,10 @@ def sync(self, parent, children):
                                             "name": "AWS_REGION",
                                             "value": f"{minio_service_region}"
                                         },
+                                        {
```

Comment:
shouldnt the export for AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY be removed now?

Reply:
why is that? I believe meghna had added those, but we are still maintaining the existing static credentials approach in this release.

Reply:
We should separate this or make this conditional